tests/inverted-index-test.js

```diff
@@ -1,49 +1,217 @@
     });
-    it('should return \'Invalid input\' for non-integers', () => {
-      expect('abc').toEqual('Invalid input');
+
+    it('should return \'Empty JSON\' for empty JSON type', () => {
+      expect(emptyBook).toEqual('Empty JSON');
     });
-    it('should return \'Invalid input\' for non-integers', () => {
-      expect(-9).toEqual('Invalid input');
+  });
+
+  describe('Join text and title function', () => {
+    it('should return a string with the joined text and title in lowercases', () => {
+      let newBook = [
+        {
+          'title': 'Alice in the wonder land',
+          'text': 'I am not as lazy as Alice'
+        },
+        {
+          'title': 'Lady',
+          'text': 'The lady is 50 years old. SHe has a child'
+        }
+      ];
+
+      const newTextTitle = 'alice in the wonder land i am not as lazy as alice lady the lady is 50 years old. she has a child';
+      newBook = InvertedIndex.joinTextTitle(newBook);
+      expect(newTextTitle).toEqual(newBook);
+    });
+  });
+
+  describe('Remove duplicate words function', () => {
+    it('should return only unique words in an array', () => {
+      let getWords =
+        ['broken',
+          'colour',
+          'crayons',
+          'go',
+          'go',
+          'he',
+          'he',
+          'is',
+          'is',
+          'poem',
+          'premier',
+          'queen',
+          'still',
+          'to',
+          'to',
+          'up',
+          'up',
+          'yes'];
+
+      const uniqueWords =
+        ['broken',
+          'colour',
+          'crayons',
+          'go',
+          'he',
+          'is',
+          'poem',
+          'premier',
+          'queen',
+          'still',
+          'to',
+          'up',
+          'yes'];
+
+      getWords = InvertedIndex.removeDuplicates(getWords);
+      expect(uniqueWords).toEqual(getWords);
+    });
+  });
+
+  describe('Tokenize function', () => {
+    it('should return in an array tokens for each file', () => {
+      // Parameter to be used to test for correctness of the tokenize method
+      let newBook = [ 
+        {
+          'title': 'Alice in Wonderland',
+          'text': 'I am not as lazy as Alice'
+        },
+        {
+          'title': 'Lady',
+          'text': 'The lady is 50 years old. SHe has a child'
+        }
+      ];
+
+      // Expected result after newBook is passed into tokenize function
+      const result =
+        ['50',
+          'a',
+          'alice',
+          'am',
+          'as',
+          'child',
+          'has',
+          'i',
+          'in',
+          'is',
+          'lady',
+          'lazy',
+          'not',
+          'old',
+          'she',
+          'the',
+          'wonderland',
+          'years'];
+
+      newBook = InvertedIndex.tokenize(newBook);
+      expect(result).toEqual(newBook);
     });
-    it('should return \'Invalid input\' for non-integers', () => {
-      expect('day').toEqual('Invalid input');
+  });
+
```

Comment:
Can you show me any of your test for PopulateIndex that ensures index is created once JSON file has been read and ensures index is correct?